docs/Using-the-dataset-validator.md

```diff
@@ -6,7 +6,7 @@ To run the validator first go to the importer folder
 `<your_cbioportal_dir>/core/src/main/scripts/importer` 
 and then run the following command:
 ```console
-./validateData.py --help
+$ ./validateData.py --help
```

Comment:
please remove `$` or use bash(?) instead of console. The `$` makes it more difficult to copy + paste the commands when the user is trying to follow the documentation.